compiler/luci/import/src/GraphBuilder.cpp

```diff
@@ -29,10 +29,9 @@ CircleNode *GraphBuilder::build(const circle::OperatorT &op, GraphBuilderContext
 
   const std::vector<int32_t> &inputs = op.inputs;
   const std::vector<int32_t> &outputs = op.outputs;
-  const auto &tensors = context->reader()->tensors();
-  const auto &opcodes = context->reader()->opcodes();
-  auto tensors_ptr = context->reader()->tensors_ptr();
-  assert(tensors_ptr != nullptr);
+  const auto opcodes = context->reader()->opcodes();
+  const auto tensors = context->reader()->tensors();
```

Comment:
do you have to change to order two variableS?

Reply:
Yes, I will reduce this kind of changes too.

Reply:
Source order is returned.